src/number_theory/bernoulli_polynomials.lean

```diff
@@ -123,23 +122,23 @@ begin
 end
 
 open power_series
-open polynomial (aeval)
 variables {A : Type*} [comm_ring A] [algebra ℚ A]
 
 -- TODO: define exponential generating functions, and use them here
 -- This name should probably be updated afterwards
 
 /-- The theorem that `∑ Bₙ(t)X^n/n!)(e^X-1)=Xe^{tX}`  -/
-theorem exp_bernoulli_poly' (t : A) :
-  mk (λ n, aeval t ((1 / n! : ℚ) • bernoulli_poly n)) * (exp A - 1) = X * rescale t (exp A) :=
+theorem exp_bernoulli' (t : A) :
```

Comment:
```suggestion
theorem bernoulli_generating_function (t : A) :
```
without changing the namespace. It's not very precise, but at least mention generating functions that it is what this result is about.

Reply:
I changed the name as suggested. I am unsure if we should keep the `TODO` above.

Reply:
Let's keep it.